docker-compose.yaml

```diff
@@ -1,43 +1,41 @@
-version: "3.8"
+version: '3.8'
 services:
-  db:
-    container_name: postgres
-    image: postgres
-    restart: always
-    healthcheck:
-      test: ["CMD-SHELL", "pg_isready -U admin -d rps"]
-      interval: 3s
-      retries: 5
-      start_period: 30s
+  mongodb:
+    container_name: mymongo
+    image: mongo
+    restart: unless-stopped
     env_file:
-      - postgres.env
+      - mongo.env
     volumes:
-      - postgres-data:/var/lib/postgresql/data
-    ports:
-      - 5432:5432
+      - ./.docker/mongodb/data/db/:/data/db/
+      - ./.docker/mongodb/data/log/:/var/log/mongodb/
     networks:
       - backend-network
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=MONGO_USER
+      - MONGO_INITDB_ROOT_PASSWORD=MONGO_PASSWORD
+      - MONGO_INITDB_DATABASE=DB
+    ports:
+      - 27017:27017
 
   web:
-    container_name: flask
-    build: flask
+    container_name: rpsFlask
```

Comment:
could we please change this back to `flask`, the nginx conf uses `flask` and i don't want to edit the files unnecessarily